onelogin_rules/onelogin_brute_force_by_username.yml

```diff
@@ -0,0 +1,30 @@
+AnalysisType: rule
+Filename: onelogin_brute_force_by_username.py
+RuleID: OneLogin.BruteForceByUsername
+DisplayName: OneLogin Brute Force Username
+Enabled: true
+LogTypes:
+  - OneLogin.Events
+Tags:
+  - OneLogin
+Severity: Medium
+Reports:
+  MITRE ATT&CK:
+    - Brute Force:Password Guessing
+Description: A OneLogin user was denied access more times than the configured threshold.
+Threshold: 10
+DedupPeriodMinutes: 10
+Reference: https://developers.onelogin.com/api-docs/1/events/event-resource
+Runbook: Analyze the IP they came from, and other actions taken before/after.
```

Comment:
I'd maybe add "Check if this user eventually authenticated successfully" to the runbook. 10 failed logins then a success might be a normal user with a bad memory, 10,000 failed logins then a success probably means your user just got brute forced.